addons/binding/org.openhab.binding.chromecast/ESH-INF/thing/thing-types.xml

```diff
@@ -106,10 +236,209 @@
         <description>Plays a given URI</description>
     </channel-type>
 
-    <channel-type id="control">
-        <item-type>Player</item-type>
-        <label>Control</label>
-        <description>Player control</description>
+    <channel-type id="onlinestatus" advanced="true">
```

Comment:
what exactly is the "online status"? Does it tell whether Chromecast can reach the Internet? Is it at all usable if it is offline? If not, I would say that this corresponds very much to the "Thing status", so this information should best be synched with the Thing status and there would be no need for a channel.